php-checks/src/main/resources/org/sonar/l10n/php/rules/php/S2245.html

```diff
@@ -0,0 +1,28 @@
+<p>As the <code>rand()</code> function relies on a pseudorandom number generator, it should not be used for security-critical applications or for
+protecting sensitive data. In such context, the methods <code>random_int()</code> or <code>random_bytes()</code> or
+<code>openssl_random_pseudo_bytes()</code> which relies on a cryptographically strong random number generator (RNG) should be used in place.</p>
```

Comment:
Should clarify with Alex what we want do do if `openssl_random_pseudo_bytes` returns FALSE in the `crypto_strong` ref parameter

Reply:
we can't reliably trace returned value anyway, so I think it is ok to ignore this scenario (there are many other ways how you can break your crypto, adding one more rare possibility is not going to change much)

Reply:
Or maybe you have some specific pattern in mind?